andela/travel_tool_front
# #162039621 Fix file download issue on travel-readiness

#### What does this PR do?
Fixes file download issue for the travel-readiness document

#### Description of Task to be completed?
* Update downloaded file extension to use .csv
* Remove limit and page queries for export to allow for the download of all records

#### How should this be manually tested?
* Clone the repos: ``` https://github.com/andela/travel_tool_front ``` and ``` https://github.com/andela/travel_tool_back.git ```
* Checkout to the respective branches ``` bg-fix-readiness-download-extension-162039621 ``` on the ``` travel_tool_front ``` and ``` bg-modify-travel-readiness-csv-data-field-titles-162039621 ``` on the ``` travel_tool_back ```
* Start up the back-end and front-end with ``` yarn start:dev ``` and ``` yarn start ``` respectively.
* Login to the app on the front end, 
* Navigate to the ``` Dashboard ``` option and select ‘export’ on the ```Travel Readiness``` card
* You should be able to download the document in a .csv file extension.

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[162039621](https://www.pivotaltracker.com/story/show/162039621)

#### Screenshots (if appropriate)
<img width="1440" alt="screen shot 2018-11-21 at 11 03 52 am" src="https://user-images.githubusercontent.com/29521319/48834089-2c9d4780-ed7d-11e8-8218-3ba5c23ea21c.png">






#### Questions:
